Routing/Geometry.swift

```diff
@@ -37,3 +37,29 @@ public class BoundingBox: NSObject {
         super.init()
     }
 }
+
+extension Point {
+    public override func isEqual(object: AnyObject?) -> Bool {
+        guard let otherObject = object as? Point else {
+            return false
+        }
+        return self == otherObject
+    }
+}
+
+extension BoundingBox {
+    public override func isEqual(object: AnyObject?) -> Bool {
+        guard let otherObject = object as? BoundingBox else {
+            return false
+        }
+        return self == otherObject
+    }
+}
+
+func ==(lhs: Point, rhs: Point) -> Bool {
+    return lhs.x == rhs.x && lhs.y == rhs.y
```

Comment:
`Double` in swift conforms to `Equatable`, so I'm assuming it compares it with a delta. Will check the source if I can find it though.